iohk-monitoring/src/Cardano/BM/Backend/Log.lhs

```diff
@@ -53,6 +52,7 @@ import           Data.Time.Clock (UTCTime, getCurrentTime)
 import           Data.Time.Format (defaultTimeLocale, formatTime)
 import           GHC.Conc (atomically)
 import           GHC.IO.Handle (hDuplicate)
+import           Lens.Micro ((^.))
```

Comment:
used in only one place! we could do without and reduce the dependencies further

Reply:
That's also fine for me. I didn't do it in this PR to keep the changes minimal and because in other repositories we already depend on `microlens` (e.g., `cardano-ledger-specs`).

Reply:
I have pushed a commit that even removes the dependency on `microlens`.

Note that this doesn't really reduce the dependency footprint in terms of compile time. The operator `(^.)` is used here to read fields from Katip's [`LogEnv`](http://hackage.haskell.org/package/katip-0.8.4.0/docs/Katip.html#t:LogEnv) record. Since `katip` itself uses `microlens` already, no extra dependencies are compiled.

You could argue that eliminating the use of lenses entirely from this repository makes sure that nobody working on this repository will be confronted with the lenses DSL, which is a whole language on its own.

I'm happy to squash the second commit into the first.

Reply:
BTW, when https://github.com/input-output-hk/cardano-ledger-specs/commit/4221bb6921159d355bd92ef00bbd763e73638a8c gets propagated to `cardano-node`, the use of `lens` in this repo will be the final use of `lens` in `cardano-node`.